checkstyle-tester/my_check.xml

```diff
@@ -26,11 +26,23 @@
          <module name="ThrowsCount">
              <property name="max" value="20000000"/>
          </module>
+         <!-- uncomment to suppress javadoc parsing errors
+         <module name="SuppressionXpathSingleFilter">
```

Comment:
I believe @romani wants this uncommented. This is why we try to make this as specific as possible.
@romani confirm

Reply:
good idea.
Filters are quick, so it will not be performance problems, but it will work as required out of the box.